Sources/Apollo/GraphQLFile.swift

```diff
@@ -41,48 +41,19 @@ public struct GraphQLFile {
                originalName: String,
                mimeType: String = GraphQLFile.octetStreamMimeType,
                fileURL: URL) {
-    guard let inputStream = InputStream(url: fileURL) else {
-      return nil
-    }
-
-    guard let contentLength = GraphQLFile.getFileSize(fileURL: fileURL) else {
-      return nil
+    guard let fileData = try? Data(contentsOf: fileURL) else {
```

Comment:
I think I was thinking more hang on to the `fileURL` - otherwise this could cause an absoltutely enormous file to get stored in memory